scripts/package

```diff
@@ -69,14 +69,3 @@ if [ ! -d $CHART_REPO_DIR ]; then
     rm -f $INDEX_PATH $CHART_REPO_DIR/assets/index.yaml
 fi
 
-if [ ${ARCH} == amd64 ]; then
-    # Move this out of ARCH check for local dev on non-amd64 hardware.
-    TAG=$TAG REPO=${REPO} go run ../pkg/image/export/main.go $SYSTEM_CHART_REPO_DIR $CHART_REPO_DIR/assets $IMAGE $AGENT_IMAGE $RUNTIME_IMAGE
```

Comment:
Isn't this still needed for generating image text files?

Reply:
good catch!